nginx_stage/lib/nginx_stage.rb

```diff
@@ -145,6 +145,8 @@ def self.clean_nginx_env(user:)
       "RAILS_LOG_TO_STDOUT" => "true",
       # name change here because only OOD_* from apache is allowed through sudo rules
       "ALLOWED_HOSTS" => ENV['OOD_ALLOWED_HOSTS'],
+      # set the duplicate to keep clean_nginx_env impepodent
```

Comment:
I can't seem to add the change myself, but should that read idempotent? (hard to say and spell).